ui/app/routes/clients.js

```diff
@@ -13,11 +13,6 @@ import classic from 'ember-classic-decorator';
 @classic
 export default class ClientsRoute extends Route.extend(WithForbiddenState) {
   @service store;
-  @service system;
-
-  beforeModel() {
-    return this.get('system.leader');
-  }
```

Comment:
As best as I can tell, this was accidentally left in here by copying the servers route page ~9 years ago.

Reply:
![nomad-bushes](https://github.com/user-attachments/assets/4c777b49-8e65-4d64-8249-5d8befcc6525)